src/decoratedSensor.ts

```diff
@@ -59,12 +65,35 @@ export default class DecoratedSensor {
 
     if(data.AQI === undefined) { return "" }
 
-    // Reset thresholds and brackets since we're reporting data anyway
-    this.calculateAQIThresholds(data.AQI, true);
-    this.currentAQINotifyBracket = this.calculateAQINotifyBracket(data.AQI);
+    this.updateAQIMonitoringState(data.AQI, true);
+
     return this.formatReport(data);
   }
 
+  private updateAQIMonitoringState(AQI: number, forceReset = false) : { old: monitoringState; new: monitoringState, bracketChange: notifyBracket, changed: boolean } {
```

Comment:
Old and new states are only used for logging at this point. The bracketChange and changed flags are the relevant ones. Bracket change is whether the AQI is outside of the previous AQI boundaries (and in what direction).